src/App.jsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
```

Comment:
<img width="724" alt="Screenshot 2023-08-02 at 16 26 33" src="https://github.com/mate-academy/react_sum/assets/53228259/30538a3b-c3e2-480b-b7a9-ef887badd691">

Reply:
Ok, I'll fix it. Could you tell me where I can find this style guide?